src/SignalR/clients/ts/package.json

```diff
@@ -1,15 +1,41 @@
 {
   "name": "client-ts",
-  "version": "0.0.0",
-  "description": "Provides a root to run npm scripts from. DO NOT PUBLISH",
+  "version": "1.0.0",
```

Comment:
This is the package.json that used to be in `common` to get a common set of dependencies. This is now just working using plain old `npm` without tricks.

We can probably hoist these dependencies to the top level package.json to make them common, but that's a separate workitem.